### ISSUE TITLE
Blog template and few posts

### ISSUE THREAD

**Issue by yangwao** [kodadot/nft-gallery#6419]
Let's prepare a template for the blog layout and posts. 

The point is to have **kodadot.xyz/blog/article-name** for SEO

Let's use
- https://nuxt.com/modules/content

I'll drop the content in a separate issue soon. 
- < _ >

Generally, I think the structure of posts would be like
- headline h1
- subtitle h2 
- banner
- date (for crawl bots)
- text

Most of the stuff will be just static, so only moving a little often.

Design
---
https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=1296%3A18906&mode=design&t=hKFpfsqkFJAtI1v7-1

![image](https://github.com/kodadot/nft-gallery/assets/5887929/182629ae-f0bc-48ea-8326-e607fc2a2283)
![image](https://github.com/kodadot/nft-gallery/assets/5887929/b6c838d1-063a-48e8-8700-675f63c5401e)

**Comment by roiLeo** [comment]
I don't think this design fits in with the kodadot design system because of the rounded borders of the cards.

**Comment by yangwao** [comment]
Hey, I've added the first article we can add, I think the description could be the headline  from article
- https://github.com/kodadot/nft-gallery/issues/6432

**Comment by daiagi** [comment]
There is a snag using `@nuxt/content`

1. current version. `nuxt content 2` is supported only for `Nuxt3`

    `Nuxt 2 is supported with Content v1` https://github.com/nuxt/content


2. `nuxt-contentV1` has an [issue with pnpm](https://github.com/nuxt/bridge/issues/506)

![image](https://github.com/kodadot/nft-gallery/assets/22791238/ffbc08ba-a019-4be9-a133-550cc4fa81e8)

**Comment by roiLeo** [comment]
Hey @daiagi, I don't think we need to use NuxtContent to begin with. 
You can try to import markdown content like we do in /first-time page.

**Comment by preschian** [comment]
> 2\. `nuxt-contentV1` has an [issue with pnpm](https://github.com/nuxt/bridge/issues/506)

seems like `@nuxt/content@^1` is not supported on `nuxt-bridge`. ref: https://github.com/nuxt/modules/blob/e5f5c6e4054e6d21acdcc86d16aa645ac8c9711f/modules/content.yml#L26C15-L26C15. If the modules supported on `nuxt-bridge` should be like this https://github.com/nuxt/modules/blob/e5f5c6e4054e6d21acdcc86d16aa645ac8c9711f/modules/pinia.yml#L16-L17

one more reason to add this case to my POC docs 🤔

> 1. current version. `nuxt content 2` is supported only for `Nuxt3`

`nuxt-content` on `nuxt-3` ready on my POC https://github.com/kodadot/nft-gallery/discussions/6429#discussioncomment-6438201 hahaha 🙈
